Packs/Slack/Integrations/SlackEventCollector/SlackEventCollector.yml

```diff
@@ -27,9 +27,9 @@ configuration:
 - display: The maximum number of audit logs to fetch
   name: limit
   type: 0
-  defaultvalue: 200
+  defaultvalue: 1000
   required: false
-  additionalinfo: Maximum is 1000. You may receive more than the passed `limit` results from a given request if there are multiple logs that were recorded at the exact same time.
+  additionalinfo: Maximum is 10000.
```

Comment:
@ShahafBenYakir forgot to verify that before I pushed. do we want a limit for this param?
except for that, fixed your notes.